src/simple_regular_solve.jl

```diff
@@ -38,7 +38,7 @@ function DiffEqBase.solve(jump_prob::JumpProblem,alg::SimpleTauLeaping;
       tprev = t[i-1]
       rate(rate_cache,uprev,p,tprev)
       rate_cache .*= dt
-      counts .= pois_rand.(rate_cache,(rng,))
+      counts .= pois_rand.(Ref(rng), rate_cache)
```

Comment:
why Ref instead of tuple?

Reply:
Oh I assumed the tuple was used to avoid broadcasting and thought using `Ref` would be the standard way to achieve this.

Reply:
Yeah, I've always just used a tuple for it